docs/tutorial/maps.rst

```diff
@@ -4,63 +4,43 @@
 Maps
 ****
 
-Map objects in **sunpy** are two-dimensional data associated with a coordinate system.
-This class offers many advantages over using packages such as `astropy.io.fits` to open 2D solar data in Python.
-**sunpy** Maps be used with any image with two spatial axes that has FITS standard compliant metadata.
-There are also 19 sources (see :ref:`map` for a complete list) that provide instrument-specific information when plotting and in some cases interpret missing metadata when it is missing from the original file.
+In this section of the tutorial, you will learn about the `Map <sunpy.map.GenericMap>` object.
+Map objects hold two-dimensional data along with the accompanying metadata.
+They can be used with any two-dimensional data array with two spatial axes and FITS-compliant metadata.
+As you will see in this tutorial, the primary advantage of using a Map object over just a bare NumPy array is the ability to perform coordinate aware operations on the underlying array, such as rotating the Map to remove the roll angle of an instrument or cropping a Map to a specific field of view.
```

Comment:
```suggestion
As you will see in this tutorial, the primary advantage of using a Map object over just a bare NumPy array is the ability to perform coordinate aware operations on the underlying array, such as rotating the Map to remove the roll angle of an instrument, cropping a Map to a specific field of view, or reprojecting from one observer to another.
```

Reply:
This tutorial actually does not (currently) cover anything about reprojection so I don't think it makes sense to include this in the tutorial summary.